docker/Dockerfile

```diff
@@ -9,7 +9,7 @@ ENV \
 RUN apt-get update && \
 	apt-get install -y git sudo software-properties-common
 
-RUN sudo npm install -g --unsafe-perm=true --allow-root backstopjs@${BACKSTOPJS_VERSION}
+RUN sudo npm install -g --unsafe-perm=true --allow-root https://github.com/justyna-olszak-wttech/BackstopJS.git
```

Comment:
Unfortunately we want to stay away from loading private libraries like this.

Reply:
Yes, I was pretty sure it will not meet acceptance, but this was necessary to make fork use my code within Docker build.